api/v1alpha1/packagebundle_types.go

```diff
@@ -104,7 +105,8 @@ type VersionConfiguration struct {
 	Required bool `json:"required"`
 
 	// Default is the name of the configuration
-	Default string `json:"default"`
+	// +kubebuilder:validation:Optional
+	Default *apiextensions.JSON `json:"default,omitempty"`
```

Comment:
This looks real interesting although I'm a bit concerned about not having much time to mess with this before our release (which in theory should be out already)